metrics-util/src/registry/mod.rs

```diff
@@ -47,7 +47,6 @@ type RegistryHashMap<K, V> = HashMap<K, V, BuildHasherDefault<RegistryHasher>>;
 /// `Registry` is optimized for reads.
 pub struct Registry<K, S>
 where
-    K: Hashable,
     S: Storage<K>,
 {
     counters: Vec<RwLock<RegistryHashMap<K, S::Counter>>>,
```

Comment:
Out of curiosity, why are we explicitly depending on a `hashbrown::HashMap` for this type? Isn't the standard library's identical?

Reply:
`std::collections::HashMap<K, V>` is based on `hashbrown` now, yes, but we use the raw entry API, which is still not yet stabilized in the standard library.